public/app/features/datasources/components/EditDataSourceActions.tsx

```diff
@@ -11,17 +12,66 @@ interface Props {
   uid: string;
 }
 
+const allowedPluginIds = [
```

Comment:
This list appears to be duplicated in another component? Can we use a single source of truth?

Reply:
Roger that, moving them to constants